packages/@react-stately/menu/src/useMenuTriggerState.ts

```diff
@@ -46,10 +40,8 @@ export function useMenuTriggerState(props: MenuTriggerProps): MenuTriggerState
   let [focusStrategy, setFocusStrategy] = useState<FocusStrategy>(null);
 
   return {
-    isOpen, 
-    setOpen, 
-    focusStrategy, 
-    setFocusStrategy,
```

Comment:
interesting, we did run into troubles when we did this ourselves in combobox, was that the reason for this change?

Reply:
Nah, these were unused and redundant with the `open`/`close`/`toggle` methods.